en/guide/xml_schema.md

```diff
@@ -78,26 +84,36 @@ As a concrete example, the definition of the [LANDING_TARGET_TYPE](../messages/c
     </entry>
 ```
 
-The main `enum` tags/fields are:
-* `name`: The name of the enum (mandatory). This is a string of capitalized, underscore-separated words.
-* `description` (optional): A string describing the purpose of the enum
-* `entry` (optional): An entry (zero or more entries can be specified for each enum)
-* [deprecated](#deprecated) (optional): A tag indicating that the enum is deprecated.
-
 
 #### entry {#entry}
 
-The enum `entry` tags/fields are:
+The "normal" enum `entry` tags/fields are:
 * `name`: The name of the enum value (mandatory). This is a string of capitalized, underscore-separated words.
 * `value` (optional): The *value* for the entry (a number).
 * `description` (optional): A description of the entry.
-* [param](#param) (optional): A param value used in a [MAV_CMD](../messages/common.md#MAV_CMD) enum entry.
 * [deprecated](#deprecated) / [wip](#wip) (optional): A tag indicating that the enum is deprecated or "work in progress". 
 
+[MAV_CMD](../messages/common.md#MAV_CMD) entries may additionally define these tags/fields:
+* [param](#param) (optional): A param value.
+* `hasLocation` (optional): A boolean (default true) indicating the entry has lat/lon/alt location information in its param values. 
+* `isDestination` (optional):  A boolean (default true) indicating that the entry is a destination.
```

Comment:
What is `isDestination` for? Isn't hasLocation the same?

Reply:
Is Destination means that the vehicle will move to/through that point.
Please read the PR comments it has been discussed in there.

Reply:
`hasLocation` (optional): A boolean (default true) indicating the entry has lat/lon/alt location information in its param 5, 6, and 7 values repectively.